src/main/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceImpl.java

```diff
@@ -231,4 +215,15 @@ public PrivilegeSummary getLatestBookingIEPSummaryForOffender(final String offen
         Optional.ofNullable(booking).orElseThrow(EntityNotFoundException.withId(offenderNo));
         return bookingService.getBookingIEPSummary(booking.getBookingId(), withDetails);
     }
+
+    @Override
+    public ResponseEntity<List<TransactionHistoryDto>> getTransactionsHistory(final String prisonId, final String nomsId,
+                                                        final String accountCode,
+                                                        final LocalDate fromDate, final LocalDate toDate) {
```

Comment:
This logic normally goes in a service.

Reply:
sure, will make the service class expose Dto objects then